assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,146 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav role="navigation" aria-label="Page navigation">
```

Comment:
Yeah that's true. I copied it from gov.uk example and left it in because I thought older screen reader users would benefit from having `role="navigation"`.